examples/with-emotion/pages/index.js

```diff
@@ -69,17 +69,14 @@ const Animated = styled.div`
 
 export default () => {
   return (
-    <>
-      <Head>
-        <title>With Emotion</title>
-      </Head>
-      <div>
-        <Basic>Cool Styles</Basic>
-        <Combined>
+    <>'     '<Head>
```

Comment:
This looks weird

Reply:
Ugh no idea why I committed these. Something went wrong with `standard --fix`. What's the best way for me to get rid of these changes?

Reply:
@timneutkens anyway I can fix these unnecessary commits. The hooks weren't allowing me to commit unless I ran `standard --fix`.

Reply:
@notrab you likely don't need to run `standard --fix` specifically. You could make these look normal, and then manually make them fit [standard's](https://github.com/standard/standard) styles.